src/options/containers/import/index.jsx

```diff
@@ -1,17 +1,27 @@
 import React from 'react'
-import { Link } from 'react-router'
+import { connect } from 'react-redux'
+import { ourState } from '../../selectors'
+import { stopLoadingHistory } from '../../actions'
+import ImportComponent from '../../components/import'
+import importHistory from '../../../browser-history/background/import-history'
 
-import { routeTitle } from '../../base.css'
 
-class ImportContainer extends React.Component {
-    render() {
-        return (
-            <div>
-                <h1 className={routeTitle}>Import</h1>
-                <p>Import specific components to come in here.</p>
-            </div>
-        )
+const mapStateToProps = state => ({
+    loadingHistoryStatus: ourState(state).loadingHistoryStatus,
+    searchIndexRebuildingStatus: ourState(state).searchIndexRebuildingStatus
+})
```

Comment:
We should use stateless components instead of `React.Component` where it's possible ([more info](https://hackernoon.com/react-stateless-functional-components-nine-wins-you-might-have-overlooked-997b0d933dbc#.u19bbm6em))